src/applications/check-in/components/LanguagePicker.jsx

```diff
@@ -36,17 +24,38 @@ function LanguagePicker() {
           label: 'Español',
           lang: 'es',
         },
-      ].map((link, i) => (
-        <button
-          key={i}
-          onClick={changeLanguage}
-          className={buttonClass(link.lang)}
-          data-testid={`translate-button-${link.lang}`}
-          type="button"
-          value={link.lang}
-        >
-          {link.label}
-        </button>
+        {
+          label: 'Tagalog',
+          lang: 'tl',
+        },
+      ].map((link, i, links) => (
+        <Fragment key={i}>
+          {language === link.lang ? (
```

Comment:
Unfortunately I think we need to keep the `startsWith` logic or something like it from before. When I go to the app with my browser language set to Spanish (Latin America) the app loads in Spanish, but the "Español" link is not shown as active. (And looks like we should add a comment explaning the reason for it too :-)

Reply:
![Screen Shot 2022-04-22 at 11 25 37 AM](https://user-images.githubusercontent.com/101649/164765847-1aadb317-0cab-4300-bb08-2db04c5a7514.png)

Reply:
Ah good catch, I updated to use `startsWith` and added a comment. I did it with out the dash since all lang codes are two characters, so just `startsWith(lang)` works for all sub lang codes